_pages/peer_tutoring.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: doc
+title: Peer Tutoring
+---
+
+## Getting Help
+
+We hold peer tutoring sessions in the basement of Siebel in the computer labs. You can join by adding yourself onto the [Queue](https://edu.cs.illinois.edu/queue/) (the schedule can be found below).
+
+Please note that this is not CS 225, so [we]({% link _pages/staff_hall_of_fame.html %}) will not do your assignments. This is one of the last classes you will take before being thrown into 400-level courses, where the assignments are much harder, help is much more limited, and hand holding is negative.
+
+> Give a student the answer and you feed them for a day; teach a student to find their own answers and you feed them for a lifetime.
+
+These peer tutoring sessions will focus on diagnosing misconceptions and holes in your understanding of the problem. This is a fairly conceptual class where your understanding of the problem will directly impact how you can implement a solution. If you have made it this far into the CS curriculum simply be compiling lecture notes and documentation to code, then you are going to have to make some serious changes in your work ethics. Be prepared to be asked to explain what you think your program is doing. Odds are that what you think your code is doing is not equivalent to what it is doing.
+
+This department is scaling rapidly, which is great, since computer science is an incredible field of knowledge that will take our civilization to new frontiers. Unfortunately, the number of people on course staff does not scale as fast. This means that we can not sit next to you for 45 minutes to read 800 lines of your code; this is peer tutoring and not a dinner date. We will wear stopwatches and [round robin](https://en.wikipedia.org/wiki/Round-robin_scheduling) between the students on the queue.
```

Comment:
Maybe we should remove the bit about wearing a stopwatch and change it to "use a stopwatch" in the spirit of honesty? I don't really care though

Reply:
Lol okay. Technically, I wear a watch with a stopwatch functionality.